build-tools/packages/build-tools/src/bumpVersion/gitRepo.ts

```diff
@@ -161,7 +161,7 @@ export class GitRepo {
      * Fetch Tags
      */
     public async fetchTags() {
-        return await this.exec(`fetch --tags`, `fetch tags`);
+        return await this.exec(`fetch --tags --force`, `fetch tags`);
```

Comment:
Is this change needed? Or was it just for testing?

Reply:
I consistently have to force because of the archive/0.39 tag. I've been meaning to ask @curtisman if maybe we could just delete that tag.